src/cosmology/compat/camb/__init__.py

```diff
@@ -21,66 +21,69 @@
 class Cosmology:
     """Cosmology API wrapper for CAMB *pars* and *results*."""
 
-    pars: CAMBparams
-    results: CAMBdata
+    data: CAMBdata
+    params: CAMBparams = field(init=False)
+
+    def __post_init__(self) -> None:
+        self.params = self.data.Params
```

Comment:
```suggestion

    @property
    def params(self) -> CAMBparams:
        return self.data.Params
```

Reply:
Not much point in even assigning it!

Reply:
There is: otherwise, CAMB generates a new object on every access.

Reply:
I guess we could use a `cached_property`, but at that point, why not assigning it?